docs/en/core-services/Catalog.md

```diff
@@ -110,35 +115,224 @@ Each time the configuration file is changed, you can use `alluxio table sync` to
 
 The configuration file is in JSON format, and can contain these configurations:
 
-1. Tables and partitions bypassing specification:
-
-    You can specify some tables and partitions to be bypassed from Alluxio, so that they will not be
-    cached in Alluxio, instead clients will be directed to access them directly from the UDB. 
-    This can be helpful when some tables and partitions are large, and accommodating them in the cache
-    is undesirable. An example configuration is like the following:
-
-    ```json
-    {
-      "bypass": {
-        "tables": [
-          "table1",
-          {"table": "table2", "partitions": ["table2_part1", "table2_part2"]}
-        ]
-      }
-    }
-    ```
-
-    You can specify which tables and partitions within these tables should be bypassed from Alluxio.
-    By specifying only the table name, all partitions of that table, if any, will be bypassed. 
-    Otherwise, you can specify specific partitions to bypass.
+1. Bypassing tables and partitions:
+
+   You can specify some tables and partitions to be bypassed from Alluxio, so that they will not be
+   cached in Alluxio, instead clients will be directed to access them directly from the UDB. 
+   This can be helpful when some tables and partitions are large, and accommodating them in the cache
+   is undesirable. An example configuration is like the following:
+
+   ```json
+   {
+     "bypass": {
+       "tables": [
+         "table1",
+         {"regex": "table[23]"},
+         {
+           "table": "table4", 
+           "partitions": [
+             "table4_part1", 
+             {"regex": "table4_part[23]"}
+           ]
+         }
+       ]
+     }
+   }
+   ```
+
+   You can specify which tables and partitions within these tables should be bypassed from Alluxio.
+   There are 2 ways to specify tables and partitions:
     
-    In the example above, table 1 is fully bypassed. Partition 1 and 2 of table 2 are bypassed, 
-    and any other partitions, if any, are not.
-
-> **Note:** When databases are attached, all tables are synced from the configured UDB.
-If out-of-band updates occur to the database or table and the user wants query results to reflect
-the updates, the database must be synced. See [Syncing Databases](#syncing-databases) for more
-information. 
+   1. Plain names: Use the exact name of a table or partition;
```

Comment:
> There are 2 ways to specify tables and partitions:
>    
>   1. Plain names: Use the exact name of a table or partition;
>   2. Regular expressions: Use a pattern to capture a set of table or partition names.

Since we are explaining the possible ways to specify tables and partitions, I mentioned table and partition names before and after "plain names", I think it's unambiguous that "plain names" refer to tables' and partitions' names. Plus, "plain names" is in contrast to "regular expressions" afterwards.